src/detailsDocs/DetailsDocs.js

```diff
@@ -1,56 +1,35 @@
 import React, { Component } from 'react'
-import { View, Image, StyleSheet, ScrollView, ListView } from 'react-native'
+import { View, Image, StyleSheet, ScrollView } from 'react-native'
 import Icon from 'react-native-vector-icons/MaterialIcons'
-let styles
+import Header from '../header'
 
 import {
-  List,
-  ListItem,
   Text,
-  SearchBar
 } from 'react-native-elements'
 
 const DetailsDocs = (props) => {
   return (
-    <ScrollView keyboardShouldPersistTaps="always" style={styles.mainContainer}>
-      <Text> { props.docsContent } </Text>
-    </ScrollView>
+    <View>
+      <Header />
+      <ScrollView keyboardShouldPersistTaps="always" style={styles.mainContainer}>
+        <Text> { props.docsContent } </Text>
+      </ScrollView>
+    </View>
   )
 }
 
+DetailsDocs.navigationOptions = {
+  title: '',
+  tabBar: {
+    icon: () =>(
+      null
```

Comment:
Because, I want the navigation for this screen, but I don't want this screen on the TabBar.  So, I'm using this screen as a Tab, but not showing it on the tabbar.

The person will go to this screen just only if he presses an item in the list of docs. The person cannot go to this screen just using the TabBar. This is the way I found to use the tabBar navigation but not show it on the TabBar.